src/components/LocationPage/ShareButtons.js

```diff
@@ -41,17 +42,58 @@ const InnerContent = props => {
     : imageBaseUrl +
       `states/${stateId.toLowerCase()}/chart/${chartIdentifier}/export.png`;
 
+  const downloadDate = moment().format('YYYY-MM-DD');
+
+  function makeDownloadFilename(chartIdentifier) {
+    const chartDownloadType = {
+      0: 'infection_growth',
+      1: 'posative_test_rate',
+      2: 'hospitalization_usage',
+      3: 'contact_tracing',
+      4: 'future_projections',
+    };
+    const chartType = chartDownloadType[chartIdentifier];
+    const location = countyId
+      ? `${countyId}_${stateId.toLowerCase()}`
+      : `${stateId.toLowerCase()}`;
+    return `${location}_${chartType}_${downloadDate}`;
+  }
+
+  function downloadChart(blob, filename) {
```

Comment:
Since this is a bit hacky, I'd maybe call it out in a comment.

// HACK: Add a link to the blob and immediately click it as described in _link-to-stack-overflow_